### DISCUSSION TITLE
Custom post links

### DISCUSSION THREAD

**Discussion by BenjamenSimon** [hugo-themes/toha#694] [Q&A] [answered]
Hello! I would like to have post links on the left hand side that directly link to

1) pdf files (such as Resume for instance, or a slide deck for a talk)

2) Custom html files - ie such as those generated by statistical programs such as R to be HTML versions of notebooks. For instance it just links to something like this (https://alexander-kahanek.github.io/project/hotel.html#).

To be clear I would like the link on the left hand side of the posts page that usually opens a post or category, to directly open the file when you click it. Not a post which you then click a link in.

I am happy to get a little dirty digging in the code if necessary, but will need guidance.

**Comment by hossainemruz** [comment, marked as answer]
Hi @BenjamenSimon, we will need some changes in the theme to support this features. I will take a look soon. Creating a feature request.

**Reply by hossainemruz** [reply]
Ref: https://github.com/hugo-toha/toha/issues/695

**Reply by BenjamenSimon** [reply]
Thank you, thats great! :)

**Reply by hossainemruz** [reply]
Hi @BenjamenSimon, I have added support for external link in sidebar. Update your theme to latest commit.

To add a external link, create a normal post. Keep the post body empty. Add only front-matter. This time, add `pageRef: <your desired url>` field on menu section of the front matter. Check this sample post: https://github.com/hugo-toha/guides/tree/main/content/posts/external-link

**Reply by BenjamenSimon** [reply]
Thank you very much! :)
